packages/carbon-graphs/src/less/core.less

```diff
@@ -6,7 +6,7 @@ svg:not(:root) {
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
 }
 
-.carbon-svgIcon {
+.carbon-svg-icon {
```

Comment:
So there is no file that uses carbon-svgIcon directly, right?

Reply:
Good catch, updated here: 13e5446e00945528fb9177c67ee7cc19728d9e91